documentation/golang/README.md

```diff
@@ -677,6 +677,270 @@ published coverage report dashboard such as this one:
          }
       }
 
+
+# Code Coverage with integration tests
```

Comment:
This reads like you copied it from somewhere, if so, can you add the URL of where you copied it from as a reference?

Reply:
done